backend/src/main/resources/db/migration/internal/V0.099__alter_control_units_and_contacts_table.sql

```diff
@@ -0,0 +1,6 @@
+ALTER TABLE public.control_units
+    ADD COLUMN department VARCHAR NOT NULL DEFAULT '',
```

Comment:
Partout ailleurs dans l'app on a mis des NULL quand un champ pouvait être non renseigné, donc pour garder une homogénéité je préfèrerais retirer la contrainte `NOT NULL` et le `DEFAULT ''`. Par souci d'homogénéité et aussi parce que ça peut causer des erreurs : en Postgres NULL et string vide ne sont pas équivalents, pour sélectionner les lignes avec des NULL dans une colonne il faut faire `WHERE column IS NULL` alors que pour  un string vide c'est `WHERE column = ''`, donc si ce n'est pas partout pareil ça peut devenir difficile de s'y retrouver et de faire les bonnes requêtes.

Par ailleurs il y a une table `department_areas`, peut-être mettre une clé étrangère dessus ?